.dockerignore

```diff
@@ -8,3 +8,4 @@ docker-compose.yml
 LICENSE
 *.md
 dockerhash.txt
+app/static
```

Comment:
This may affect existing installations as not everyone may have set up the nginx part.

Reply:
I may be missing something, but the static fiels aren't used at all by the server-side process, right? Which means they shouldn't be copied into that image either way

There's a separate .dockerignore for the nginx container for this reason

Reply:
I believe they are used as a fallback, but yes, they shouldn't be in the image ideally.